sheets436/src/main/java/edu/umd/cmsc436/sheets/Sheets.java

```diff
@@ -200,6 +200,7 @@ static void showGooglePlayErrorDialog(Host host, Activity hostActivity) {
         RH_POP("'Balloon Test (RH)'"),
         LH_CURL("'Curling Test (LH)'"),
         RH_CURL("'Curling Test (RH)'"),
+        FAKE("'Fake Test'"),
```

Comment:
Don't have unless enums. Users are exposed to this. Run the actual enums on new spreadsheets.